tests/gold_tests/pluginTest/client_context_dump/gold/client_context_dump.gold

```diff
@@ -0,0 +1,2 @@
+``LookupName: :/home/zeyuany/repos/dyrock-ts/tests/_sandbox/client_context_dump/ts/etc/trafficserver:/home/zeyuany/repos/dyrock-ts/tests/_sandbox/client_context_dump/ts/ssl/two.com.pem:, Subject: C=US,ST=Illinois,L=Champaign,OU=Two,O=Apache,emailAddress=ssl@two.com,CN=two.com. SAN: . Serial: . NotAfter: Jun 21 20:39:43 2019 GMT.
```

Comment:
Will in general the autest output include the name or your home directory?

Reply:
Ooops.......